git-bugreport: add page (#5097)
tldr-pages/tldr: pages/common/git-bugreport.md

# git bugreport

> > Captures information about the user's machine, Git client, and repository state, as well as a form requesting information about the behavior the user observed, into a single text file which the user can share in order to report an observed bug.

Reviewer: Perhaps we could be more concise?
```suggestion
> Captures debug information from the system and user, generating a text file to aid in the reporting of a bug in Git.
```

- Create a new bugreport file in the current directory:

`git bugreport`

> - Create a new bugreport file in the specified, creating the directory if it does not exist:

Reviewer: Perhaps this will read better?
```suggestion
- Create a new bugreport file in the specified directory, creating it if it does not exist:
```

`git bugreport --output-directory {{path/to/directory}}`

> - Create a new bugreport file with the filename suffix in the specified `strftime` format:

Reviewer: Perhaps this will read better?
```suggestion
- Create a new bugreport file with the specified filename suffix in `strftime` format:
```
